### PR TITLE
Support mapping from key to VSCode command in Cursorless keyboard mode

### LINKED ISSUE #1963: Support mapping from key to VSCode command in Cursorless keyboard mode
so you can eg map `j` to go up a line

### PR DESCRIPTION
**Pull request by pokey**
- Fixes #1963

## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [x] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

**Comment by pokey** [comment]
ok @Kn0rk it has begun 😊

File: docs/user/experimental/keyboard/modal.md [hunk 1]
```diff
@@ -97,6 +97,24 @@ To bind keys that do not have modifiers (eg just pressing `a`), add entries like
...
+    // }
+
+    // If you'd like to run the command on the active target, use the following format
+    "am": {
+      "commandId": "editor.action.joinLines",
+      "executeAtTarget": true,
+      // "keepChangedSelection": true,
+      // "exitCursorlessMode": true,
+    }
+  }
```
Comment (pokey) [inline docs/user/experimental/keyboard/modal.md:117]: @josharian @kn0rk does this make sense?

**Comment by pokey** [comment]
ok @AndreasArvidsson this one is ready for review. If you're happy, with it, let's give @kn0rk and @josharian a chance for a quick look, as they've expressed interest in keyboard stuff (and supporting arbitrary VSCode commands was actually @kn0rk's idea 😊)

**Comment by Kn0rk** [comment]
Looks good on a first glance. I will try to set it up on the weekend and give it a test drive.

**Comment by pokey** [comment]
> Looks good on a first glance. I will try to set it up on the weekend and give it a test drive.

Sounds good; I'll keep stacking PRs off of this one and give you a branch to play with this weekend that has all my changes

File: docs/user/experimental/keyboard/modal.md (hunk 1 above)
Comment (josharian) [reply docs/user/experimental/keyboard/modal.md:117]: haven't taken it for a spin but reading it it makes sense

**Pull request merged**